### PR TITLE
Feature/UI creation

### PR DESCRIPTION
**Pull request by MosDevx**
Features Implemented :

    The project is a single-page application (SPA) built with React and Redux.
    The data retrieved from the API should be stored in the Redux store.
    filter the data that you retrieve from the API using a Filter stateless component.
    Every page (the main page and the pages for each item) should have a unique route within the SPA.
    The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### CODE REVIEW

File: src/Components/PokemonList.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	return (
+		<div>
+			{/* <div className='flex'>
+					<button onClick={()=>changeType('all')}>All</button>
+					<button onClick={()=>changeType('water')}>Water</button>
+					<button onClick={()=>changeType('fire')}>Fire</button>
+
+			</div> */}
```
Comment (codepantha) [inline src/Components/PokemonList.jsx]: - [x] Please remove commented code as it clutters your codebase 👍

File: src/Components/PokemonList.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			: data ? (
+				
+				data.map((name)=>{
+					return <Pokemon key={name} name={name}></Pokemon>
+				})
+			)	: null
+		}
+
+
+		</div>
```
Comment (codepantha) [inline src/Components/PokemonList.jsx]: - [x] Please ensure to stick to the typography, layout, and spacing between cards as shown in the design guidelines below
### Design Guideline
image
### Your Implementation
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @MosDevx 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [x] Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the "ACKNOWLEDGEMENTS" section of your README.
- [x] Please implement the functionality to filter through the list of pokemon cards using any given parameter e.g the name using a `Filter` stateless component 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/PokemonList.jsx (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/Components/PokemonList.jsx]: Fixed.

File: src/Components/PokemonList.jsx (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/Components/PokemonList.jsx]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @MosDevx 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.
✔️ All Test Are Passing.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**